pkg/app/pipectl/cmd/deployment/logs.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2023 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deployment
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/pipe-cd/pipecd/pkg/app/server/service/apiservice"
+	"github.com/pipe-cd/pipecd/pkg/cli"
+)
+
+type logs struct {
+	root *command
+
+	deploymentID string
+	stageID      string
+	stdout       io.Writer
+}
+
+func newLogsCommand(root *command) *cobra.Command {
+	c := &logs{
+		root:   root,
+		stdout: os.Stdout,
+	}
+	cmd := &cobra.Command{
+		Use:   "logs",
+		Short: "Get deployment stage logs.",
```

Comment:
If this is just for getting the stage log of deployments, this subcommand name is too luxurious.
How about adding this command to the deployment sub-command?

Reply:
Hmmm
How about pipectl deployments logs stage-log?

Reply:
I feel `pipectl deployment logs` is fine. wdyt?